ledger-core/rms/any.go

```diff
@@ -55,13 +55,21 @@ func (p *Any) MarshalTo(b []byte) (int, error) {
 	if p.value != nil {
 		return p.value.MarshalTo(b)
 	}
+	_, _, err := UnmarshalType(b, func(u uint64) reflect.Type { return reflect.TypeOf(struct{}{}) })
+	if err != nil {
+		return 0, err
+	}
 	return 0, nil
```

Comment:
why do you need if here? not just 
return 0, err

Reply:
fixed